funnel/models/project.py

```diff
@@ -704,6 +704,13 @@ def location_geonameid(self):
     def permissions(self, user, inherited=None):
         perms = super(Project, self).permissions(user, inherited)
         perms.add('view')
+        if self.state.WITHDRAWN:
+            if user not in self.admin_team.users:
+                abort(410)
+            elif user not in self.review_team.users:
+                abort(410)
+            elif user not in self.checkin_team.users:
+                abort(410)
```

Comment:
`abort` is a view action. You should not be performing view actions in a model, which is only responsible for data state and data integrity checks. In this case, the `if` condition should not offer the `view` permission when the project is in withdrawn state.

Separately, since this changes data access, it needs a ticket presenting a case for (a) why a project should not be accessible in the withdrawn state, and (b) what is broken or not broken by such a change. The ticket and argument should precede this pull request.